hammer/par/openroad/__init__.py

```diff
@@ -174,7 +174,7 @@ def output_netlist_filename(self) -> str:
     @property
     def output_sim_netlist_filename(self) -> str:
         return os.path.join(self.run_dir, "{top}.sim.v".format(top=self.top_module))
-    
```

Comment:
Thanks for fixing this stuff. In the future we should use a Python formatter (e.g. black) and make that part of the CI check.

Reply:
Ah. This is just done with my Vim config...